app/controllers/api/v1/wines_controller.rb

```diff
@@ -12,10 +12,13 @@ def fetch_all_the_things(wine_api_id)
   formatted_wine_data = format_wine_data(parsed_response[:data])
 
   # Need to get weather info
-  formatted_weather_data = fake_weather_data
+
+  weather_data = WeatherFacade.fetch_weather(wine_vintage, wine_region)
+  
+#   formatted_weather_data = fake_weather_data
 
   # Need to combine the two (and return the value)
-  combine_data(formatted_wine_data, formatted_weather_data)
+  combine_data(formatted_wine_data, weather_data)
```

Comment:
This was changed per the merge conflict and based on what we are now expecting from `weather_data`. `formatted_weather_data` was what Alex originally had when mocking the weather data.